corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -339,8 +387,220 @@ def _get_table_size(config):
             for datasource in static_datasources + dynamic_datasources
         ], key=lambda r: r[-1] if r[-1] != 'Table not found' else 0)
 
+        resource_model['datasources_info'] = rows
+
+    def collect_case_transactions(self):
+        db_name = get_db_aliases_for_partitioned_query()[0]
+        db_cursor = connections[db_name].cursor()
+
+        with db_cursor as cursor:
+            cursor.execute("""
+                SELECT COUNT(*) as num_forms, d.count as num_updates
+                FROM (
+                    SELECT COUNT(*) as count
+                    FROM form_processor_casetransaction
+                    WHERE case_id IN (
+                        SELECT case_id
+                        FROM form_processor_commcarecasesql
+                        WHERE domain = %s
+                    )
+                    GROUP BY form_id
+                ) AS d
+                GROUP BY d.count;
```

Comment:
Grouping by a count seems odd, why not just sum the values which is what you're doing below anyway.
```
select count(*) as num_forms, sum(d.count) as num_transactions from (....) as d;
```

Reply:
Well spotted!